### PR TITLE
Skip over frames dynamically instead of with an index

### PR DESCRIPTION
**Pull request by roncohen**
Depends on https://github.com/hallas/stacko/pull/2.

### CODE REVIEW

File: opbeat.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
-	"github.com/hallas/stacko"
+	"github.com/roncohen/stacko"
```
Comment (hallas) [inline opbeat.go]: Is this on purpose?

Comment (roncohen) [inline opbeat.go]: Only until you merge my PR ;)

File: opbeat.go [hunk 2]
```diff
@@ -138,6 +139,12 @@ func New(organizationID, appID, secretToken string) *Opbeat {
 	opbeat.LoggerName = "default"
 	opbeat.Logger = log.New(os.Stderr, "", log.LstdFlags)
 
+	// Get the current package name to be used in skipping frames later
```
Comment (hallas) [inline opbeat.go]: Interesting solution!

**Comment by hallas** [comment]
It should build now. I am ok with merging this :-)

**Comment by roncohen** [comment]
Awesome!

**Pull request merged**